frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="text-left">
+    <BreadcrumbsNav />
+    <h1 class="text-h6 mt-n2 mb-2 font-weight-bold">
+      Les cantines de mon territoire
+    </h1>
+    <p>Établissements dans {{ departmentsString }}</p>
+    <v-data-table
+      :footer-props="footerProps"
+      :items="visibleCanteens"
+      :options.sync="options"
+      :loading="loading"
+      :headers="headers"
+      :server-items-length="canteenCount || 0"
+    >
+      <template v-slot:[`item.publicationStatus`]="{ item }">
+        <router-link
+          v-if="item.publicationStatus === 'published'"
+          :to="{ name: 'CanteenPage', params: { canteenUrlComponent: `${item.id}--${item.name}` } }"
+        >
+          Publiée
+        </router-link>
+        <span v-else>
+          Non publiée
+        </span>
+      </template>
+    </v-data-table>
+  </div>
+</template>
+
+<script>
+import jsonDepartments from "@/departments.json"
+import { getObjectDiff } from "@/utils"
+import BreadcrumbsNav from "@/components/BreadcrumbsNav"
+
+export default {
+  name: "ElectedCanteens",
+  components: { BreadcrumbsNav },
+  data() {
+    return {
+      loading: false,
+      canteenCount: null,
+      visibleCanteens: [],
+      limit: 10,
+      options: {
+        sortBy: [],
+        sortDesc: [],
+        page: 1,
+      },
+      footerProps: {
+        disableItemsPerPage: true,
+        itemsPerPageText: "Cantines per page",
+      },
+      headers: [
+        { text: "SIRET", value: "siret", sortable: true },
+        { text: "Nom", value: "name", sortable: true },
+        { text: "Ville", value: "city", sortable: true },
```

Comment:
I'd wait for a second round before adding it, mainly bc there are a few different options/considerations. Maybe we would like to add a filtering system like the one on nos-cantines. Also the department column would be dynamic since many elected officials will only have one department they look after.